template/sync_template.py

```diff
@@ -0,0 +1,68 @@
+"""This script serves as a skeleton template for synchronous AgentQL scripts."""
+
+import logging
+
+import agentql
+from agentql.ext.playwright.sync_api import Page
+from playwright.sync_api import sync_playwright
+
+# Set up logging
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger(__name__)
+
+# Set the URL to the desired website
+URL = "WEBSITE_URL"
```

Comment:
we can highlight placeholders

```suggestion
URL = "<Replace with the correct url>"
```